tools/Environments/DevHome.Environments/ViewModels/OperationsViewModel.cs

```diff
@@ -46,12 +51,22 @@ public partial class OperationsViewModel : IEquatable<OperationsViewModel>
 
     private Action? DevHomeAction { get; }
 
-    public OperationsViewModel(string name, string icon, Func<string, Task<ComputeSystemOperationResult>> command, ComputeSystemOperations computeSystemOperation)
+    private WinUIEx.WindowEx? _windowEx;

```

Comment:
The ViewModel can be made with DI.  That way windowEx does not need to be passed down the call chain.  

Does not need to be changed here because the constructor would need to change, and that would affect way too much code.